native_client/java/Makefile

```diff
@@ -12,8 +12,16 @@ clean: apk-clean
 apk-clean:
 	./gradlew clean
 
-apk: apk-clean bindings
-	LIBDEEPSPEECH_SO=$(LIBDEEPSPEECH_SO) ./gradlew build
+libs-clean:
+	rm -fr libdeepspeech/libs/*/libdeepspeech.so
+
+libdeepspeech/libs/%/libdeepspeech.so:
+	mkdir libdeepspeech/libs/$*/ || true
```

Comment:
Looks like the change suggested by @9define is still applicable here, prefixing the command with a `-` instead of using the `|| true` idiom.